std-bits/table/src/main/java/org/enso/table/data/column/operation/map/text/StringStringLongOp.java

```diff
@@ -0,0 +1,75 @@
+package org.enso.table.data.column.operation.map.text;
+
+import org.enso.table.data.column.builder.StringBuilder;
+import org.enso.table.data.column.operation.map.BinaryMapOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemAggregator;
+import org.enso.table.data.column.storage.SpecializedStorage;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.column.storage.StringStorage;
+import org.enso.table.data.column.storage.numeric.LongStorage;
+import org.enso.table.data.column.storage.type.TextType;
+import org.enso.table.error.UnexpectedTypeException;
+import org.graalvm.polyglot.Context;
+
+public abstract class StringStringLongOp<T>
```

Comment:
```suggestion
public abstract class StringStringLongOp
```

Is the generic `<T>` parameter used anywhere? I cannot see it.

Reply:
Also the name of the op sounds to me like
```java
protected abstract long doString(String a, String b);
```

Maybe it could be `StringLongToStringOp` or something like that?

Reply:
Fixed